.drone.yml

```diff
@@ -896,4 +896,43 @@ volumes:
 trigger:
   event:
   - promote
+---
+kind: pipeline
+type: docker
+name: dispatch
+
+platform:
+  os: linux
+  arch: amd64
+
+steps:
+  - name: dispatch
+    image: rancher/dapper:v0.5.0
```

Comment:
This is a pretty old version, the rest of the file appears to use 0.5.2, is there a specific reason to use this version?

Reply:
no, will update